Source/GSStoryboardLoader.m

```diff
@@ -0,0 +1,218 @@
+/* <title>GSXibLoader</title>
+
+   <abstract>Xib (Cocoa XML) model loader</abstract>
+
+   Copyright (C) 2010, 2011 Free Software Foundation, Inc.
+
+   Written by: Fred Kiefer <FredKiefer@gmx.de>
+   Created: March 2010
+
+   This file is part of the GNUstep Base Library.
+
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02110 USA.
+*/
+
+#import <Foundation/NSArray.h>
+#import <Foundation/NSAutoreleasePool.h>
+#import <Foundation/NSData.h>
+#import <Foundation/NSDebug.h>
+#import <Foundation/NSDictionary.h>
+#import <Foundation/NSException.h>
+#import <Foundation/NSFileManager.h>
+#import <Foundation/NSKeyedArchiver.h>
+#import <Foundation/NSKeyValueCoding.h>
+#import <Foundation/NSString.h>
+#import <Foundation/NSValue.h>
+
+#import "AppKit/NSApplication.h"
+#import "AppKit/NSMenu.h"
+#import "AppKit/NSNib.h"
+#import "GNUstepGUI/GSModelLoaderFactory.h"
+#import "GNUstepGUI/GSNibLoading.h"
+#import "GNUstepGUI/GSXibLoading.h"
+#import "GNUstepGUI/GSXibKeyedUnarchiver.h"
+
+@interface NSApplication (StoryboardCompatibility)
+- (void) _setMainMenu: (NSMenu*)aMenu;
+@end
+
+@interface NSMenu (StoryboardCompatibility)
+- (BOOL) _isMainMenu;
+@end
+
+@implementation NSMenu (StoryboardCompatibility)
+
+- (BOOL) _isMainMenu
```

Comment:
Now we have this method multiple times. Better move it to NSMenu.m and have it just once,

Reply:
See above.

Reply:
Class removed.